### DISCUSSION TITLE
Implement '/' based block editor interface for portable-text

### DISCUSSION THREAD

**Discussion by somarand** [sanity-io/sanity#3618] [Sanity Studio Support]
More and more apps now use the '/' based block editor interface. Its a huge UX improvement to the old wysiwyg  click and type interface. Being a structured content editor its a no brainer its included with sanity. Thanks

![notion-shortcuts-commands](https://user-images.githubusercontent.com/113043435/189378534-7a72c141-ecb8-481e-a6b0-9f76a60a4214.png)
[Notion example](https://www.notion.so/)

**Comment by ghost** [comment]
This would be great to have. Right now the actual task of creating the body content which is often done with the portable text is not different from the old Wordpress days. Block editors like Notion and now Gutenberg is way better. I love how editorjs does this, they have a great UI. Perhaps we can do a custom input component

**Comment by cloudjareen** [comment]
Agree with @somarand +1
